modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf-cdi/beanValidation.mustache

```diff
@@ -0,0 +1,53 @@
+{{#required}}
```

Comment:
OK, the only difference is that PathParams are always required, so they don't need a check for NotNull, I commented this in beanValidationPathParams.mustache. So we could extract all other annotation checks into a beanValidationCommonParams.mustache.

Regarding cross-language templates using lambdas this is a nice feature of Mustache, but we need green light from @fehguy /@wing328 for cross-language template reuse.